clang/cmake/caches/Fuchsia.cmake

```
@@ -165,33 +175,57 @@ endif()
 set(BOOTSTRAP_LLVM_ENABLE_LLD ON CACHE BOOL "")
 set(BOOTSTRAP_LLVM_ENABLE_LTO ON CACHE BOOL "")
 
-set(_FUCHSIA_BOOTSTRAP_TARGETS
-  check-all
-  check-clang
-  check-lld
-  check-llvm
-  check-polly
-  llvm-config
-  clang-test-depends
-  lld-test-depends
-  llvm-test-depends
-  test-suite
-  test-depends
-  toolchain-distribution
-  install-toolchain-distribution
-  install-toolchain-distribution-stripped
-  install-toolchain-distribution-toolchain
-  clang)
-
-if(FUCHSIA_ENABLE_LLDB)
-  list(APPEND _FUCHSIA_ENABLE_PROJECTS lldb)
-  list(APPEND _FUCHSIA_BOOTSTRAP_TARGETS
-    check-lldb
-    lldb-test-depends
-    debugger-distribution
-    install-debugger-distribution
-    install-debugger-distribution-stripped
-    install-debugger-distribution-toolchain)
+if(FUCHSIA_ENABLE_PGO)
+  set(BOOTSTRAP_LLVM_BUILD_INSTRUMENTED ON CACHE BOOL "")
+
+  set(_FUCHSIA_BOOTSTRAP_TARGETS
+    generate-profdata
+    stage2
+    stage2-toolchain-distribution
+    stage2-install-toolchain-distribution
+    stage2-install-toolchain-distribution-stripped
+    stage2-install-toolchain-distribution-toolchain
+    stage2-check-all
+    stage2-check-lld
+    stage2-check-llvm
+    stage2-check-clang
```

```suggestion
    stage2-check-clang
    stage2-check-poly
```
Seems to be missing on a dry-run of this patch. https://ci.chromium.org/ui/p/fuchsia/builders/toolchain.ci.shadow/clang-linux-x64/b8720396545015029937/infra